README.rst

```diff
@@ -48,23 +49,14 @@ Testing
 -------
 There are some unit and integration tests in the tests directory.
 
+Running tests depends on some additional packages: python3-pytest,
+python3-hypothesis.
+
 These can be run by setting your PYTHONPATH environment variable to the CLI
 src directory, setting the STRATIS environment variable to the absolute path
-of your stratisd executable, and entering::
-
-   > py.test tests
-
-at the command prompt. Entering::
-
-   > tox
-
-will cause some tests to be run in the tox virtual environment. Integration
-tests, which requires the CLI to connect with the Stratis daemon, will not
-be run.
+of your stratisd executable, for example::
```

Comment:
The only tests tox still runs are lints, which are really more checks. But I wouldn't choose to stop mentioning it entirely.